### DISCUSSION TITLE
Welcome to OpenOOD Discussions!

### DISCUSSION THREAD

**Discussion by Jingkang50** [Jingkang50/OpenOOD#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by romain-xu-darme** [comment]
Hello !
I'm currently trying to reproduce some of the results of you paper and encountered some small discrepancies between the code and the paper: for both CIFAR10 and CIFAR100, far OoD datasets are [MNIST, FMNIST, texture and place365] in the paper / [MNIST, SVHN, texture, place365] in the code. Which datasets have been used to generate the table in p8 of your paper? Thanks a lot for sharing this code :)

**Reply by Jingkang50** [reply]
Ah, we should follow the result here, which yields the main result of the OpenOOD paper. https://docs.google.com/spreadsheets/d/1gGHpdA3sSgfGpsrDUgt9lejvIbf5yOrDyJ511zsn1uY/edit?usp=sharing
Sorry for the inconvenience it made..
